include/rogue/GeneralError.h

```diff
@@ -33,10 +33,11 @@ namespace rogue {
     * in the system.
     */
    class GeneralError : public std::exception {
-         char text_[200];
+         char text_[600];
```

Comment:
Should "600" be defined as a constant in a header file?  It appears to get used again GeneralError.cpp

Reply:
I knew one of you guys would go after that :). I will do it properly.